src/types/Behov.ts

```diff
@@ -38,3 +38,10 @@ export enum Behovfelt {
     ArbeidsmiljøBehov = 'arbeidsmiljøBehov',
     GrunnleggendeBehov = 'grunnleggendeBehov',
 }
+
+export interface AlleBehov {
```

Comment:
Har trukket ut `AlleBehov` til et eget interface og kombinerer denne med `KandidatUtenBehov` i Kandidat.ts for å få en `Kandidat`. Har unngått å bruke `[Behovfelt.DittenOgDatten]` i definisjonen her slik at vi får autocomplete på kandidat.<behov>